### ISSUE TITLE
Enable / disable continuous payments

### ISSUE THREAD

**Issue by tselit** [interledger/web-monetization-extension#238]
When the extension is connected to a wallet:

1. A user can enable the extension to make continuous WM payments to any website by setting the toggle like this, this will apply the configured hourly rate of WM payments:
![Image](https://github.com/interledger/web-monetization-extension/assets/90185897/ae2a119b-7aa5-422c-b441-35b4b091bc36)

2. A user can disable the extension from making any continuous WM payments to all WM websites by setting the toggle like this:
![Image](https://github.com/interledger/web-monetization-extension/assets/90185897/eba649d2-608b-4ee3-9c68-8b336cfe7833)

**Comment by tselit** [comment]
**We have been discussing a possible issue**: what is the difference or relationship between the functionality of enable/disable continuous payments compared to the functionality of enable/disable WM globally. 

**One of the questions is**: does this enable/disable continuous payments apply only the current website or does it apply to any website I visit? (probably not a right/wrong way, just an implementation decision for us to make)

**Comment by tselit** [comment]
Agreed to put this ticket on-hold until we (or Tseli) have a full conversation with Alex about what the distinction is between en/disabling WM globally or the use of continuous payments when the Exceptions List functionality is implemented.

**Comment by tselit** [comment]
Issue **unblocked** by decision agreed with Alex today (14 May):
- We realized that there is no difference between the intended way that the global WM en/disable toggle works, and how the "continuous payments" toggle is supposed to work.
- **Decision**: hide the global enable/disable toggle (because it duplicates the intended behaviour of the continuous payments toggle).
- **Decision**: the "continuous payments" toggle should implement the functionality that had been implemented for #61 (i.e. the way we handled the global enable/disable WM toggle).

Here is the updated UI:
![image](https://github.com/interledger/web-monetization-extension/assets/90185897/debac493-98b5-46ce-bbee-b0a85ed92a08)
